spec/features/tasks/nomeclature/browse_spec.rb

```diff
@@ -6,8 +6,10 @@
     before { sign_in_user_and_select_project}
 
     context 'when I visit the task page', js: true do
-      let!(:root) { @project.send(:create_root_taxon_name) } 
+      let!(:root) { @project.send(:create_root_taxon_name) }
       let!(:genus) { Protonym.create!(by: @user, project: @project, parent: root, name: 'Aus', rank_class: Ranks.lookup(:iczn, :genus)) }
+      let!(:genus_synonym) { Protonym.create!(by: @user, project: @project, parent: root, name: 'Bus', rank_class: Ranks.lookup(:iczn, :genus)) }
```

Comment:
@LordFlashmeow Can you move these new lines to the most pertinent context?   I.e. don't create when we don't need them (have no specs to test their values).